aws-lc-rs/src/digest/sha.rs

```diff
@@ -84,6 +96,23 @@ pub static SHA1_FOR_LEGACY_USE_ONLY: Algorithm = Algorithm {
     hash_nid: NID_sha1,
 };
 
+/// SHA-256 as specified in [FIPS 180-4].
+///
+/// [FIPS 180-4]: http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.180-4.pdf
+#[allow(deprecated)]
+pub static SHA224: Algorithm = Algorithm {
+    output_len: SHA224_OUTPUT_LEN,
+    chaining_len: SHA224_OUTPUT_LEN,
+    block_len: 512 / 8,
```

Comment:
Nit: It's not clear why we don't have a constant defined for this value like we do for the other parameters. (The same in SHA256.)